File: pytools/data/data.code.php.client/ReqChatGm.php
<?php
namespace protocol;

use network\Packet;


class ReqChatGm
{
	private $content = '';


	public function encode()
	{
		$packet = new Packet();
		$packet->writeString($this->content);
		return $packet->encode(Msg::$P_REQ_CHAT_GM);
	}


	public function setContent($content)
	{
		$this->content = $content;
	}

}

File: pytools/data/data.code.csharp.client/ReqChatGm.cs
using System.Collections;
using System.Collections.Generic;


public class ReqChatGm
{
	private string _content;


	public Packet Encode()
	{
		Packet packet = new Packet();
		packet.WriteString(this._content);
		packet.Encode(Msg.P_REQ_CHAT_GM);
		return packet;
	}


	public string content
	{
		get { return this._content; }
		set { this._content = value; }
	}

}
